library/ajax/username_report_ajax.php

```diff
@@ -0,0 +1,306 @@
+<?php
+/**
+ * library/ajax/claims_viewer_ajax.
+ * file adapted to present user activity log
+ * Copyright (C) 2012 Medical Information Integration <info@mi-squared.com>
+ * Copyright (c) 2018 Growlingflea Software <daniel@growlingflea.com>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/mpl-license.php>;.
+ *
```

Comment:
Be consistent with the headers, plz  (Author/package)

Reply:
fixed